anyone ran into an issue where fs.exists never responds to it's call back for some reason